CMakeLists.txt

```diff
@@ -137,36 +163,45 @@ add_subdirectory(apps/EagleImport)
 
 # Add unittests
 if(BUILD_TESTS)
-    add_subdirectory(tests/unittests)
+  add_subdirectory(tests/unittests)
 endif()
 
 # Generate translation file target
 set(LIBREPCB_QM_FILES_DIR "${CMAKE_BINARY_DIR}/i18n")
 file(MAKE_DIRECTORY "${LIBREPCB_QM_FILES_DIR}")
-file(GLOB LIBREPCB_TS_FILES RELATIVE ${CMAKE_SOURCE_DIR} i18n/librepcb_*.ts)
-set_source_files_properties(${LIBREPCB_TS_FILES} PROPERTIES OUTPUT_LOCATION "${LIBREPCB_QM_FILES_DIR}")
-if (Qt5_VERSION VERSION_LESS 5.12)
-    # Qt <5.12 doesn't support OPTIONS
-    qt5_add_translation(LIBREPCB_QM_FILES ${LIBREPCB_TS_FILES})
+file(
+  GLOB LIBREPCB_TS_FILES
+  RELATIVE ${CMAKE_SOURCE_DIR}
+  i18n/librepcb_*.ts
+)
+set_source_files_properties(
+  ${LIBREPCB_TS_FILES} PROPERTIES OUTPUT_LOCATION "${LIBREPCB_QM_FILES_DIR}"
+)
+if(Qt5_VERSION VERSION_LESS 5.12)
+  # Qt <5.12 doesn't support OPTIONS
+  qt5_add_translation(LIBREPCB_QM_FILES ${LIBREPCB_TS_FILES})
 else()
-    qt5_add_translation(LIBREPCB_QM_FILES ${LIBREPCB_TS_FILES} OPTIONS -silent)
+  qt5_add_translation(LIBREPCB_QM_FILES ${LIBREPCB_TS_FILES} OPTIONS -silent)
 endif()
 add_custom_target(librepcb_translations DEPENDS ${LIBREPCB_QM_FILES})
 add_dependencies(librepcb librepcb_translations)
 
 # Install target for shared files
-file(GLOB SHARE_DIRS
-    LIST_DIRECTORIES true
-    RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
-    share/*
+file(
+  GLOB SHARE_DIRS
+  LIST_DIRECTORIES true
+  RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
+  share/*
 )
 foreach(d ${SHARE_DIRS})
-    if(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/${d}")
-        install(
-            DIRECTORY ${d}
-            DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}
-            REGEX "/\.(git|gitattributes|gitignore)$" EXCLUDE
-        )
-    endif()
+  if(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/${d}")
+    install(
+      DIRECTORY ${d}
+      DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}
+      REGEX "/\.(git|gitattributes|gitignore)$" EXCLUDE
+    )
+  endif()
 endforeach()
-install(DIRECTORY ${CMAKE_BINARY_DIR}/i18n DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/librepcb)
+install(DIRECTORY ${CMAKE_BINARY_DIR}/i18n
+        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/librepcb
+)
```

Comment:
One last thing: Why is the format of this `install` different than the one above (inside the `if`)? (In my opinion the version above looks better, but I don't have a strong opinion on that.)

Reply:
Hmm I don't know exactly - somehow this can be controlled with the configuration parameters `min_prefix_chars` and `max_prefix_chars` but I still don't understand what they do exactly so I left them at their default value :see_no_evil: 

However, we can enforce wrapping the `install` command by adding it to the `always_wrap` configuration :slightly_smiling_face:  This fixes the inconsistency here and does not change any other file. I'll update the branch accordingly.